README.md

```diff
@@ -50,37 +50,6 @@ setInterval(async () => {
 }, 60000);
```

Comment:
theres a line up near the top that says "and supports both callbacks and promises. " which needs to change